draft-ietf-scitt-architecture.md

```diff
@@ -165,6 +165,21 @@ Issuers can register their Signed Statements on one or more Transparency Service
 
 # Introduction
 
+## NEW
+This document defines an architecture, message flows, and a base set of extensible message structures to make signed data items transparent via verifiable data structures that are maintained by corresponding transparency services.
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements at a transparency service is akin to a notarization procedure.
+Transparency services therefore act like notaries that record (register) that statements were made, irrevocably.
+How these statements are managed or stored---beside the specified registration procedure---is out-of-scope of this document.
+Earlier, similar approaches exist, such as Certificate Transparency {{-CT}}.
+The SCITT approach follows a more generic paradigm than previous approaches.
+In this SCITT core architecture, the signed statements registered are opaque to the transparency services that act as a notary.
```

Comment:
```suggestion
In its core architecture, signed statements are opaque to the transparency services that act as notaries.
```

Reply:
Ah. I prefer to repeat the subject to make reference to previous sentences more unambiguous, but I can live with one "it".